src/utils/notifications.rs

```diff
@@ -3,12 +3,19 @@ use notify_rust::Notification;
 
 // Implementation to create and push new notifications
 #[allow(non_snake_case)]
-pub fn PushNotification(title: String, content: String) {
+pub fn PushNotification(title: String, content: String, notification_sound: String) {
     let summary = format!("Uplink - {}", title);
     let _n = Notification::new()
         .summary(summary.as_ref())
```

Comment:
not sure why as_ref() is being used here but this also works: `&summary` 
`content.as_ref()` could also be `&content`. 

also, i'll re-iterate here the suggestion to use an `enum` for `notification_sound`

Reply:
Thank you!

Reply:
We're using Sounds now as the type.